Automattic/sensei
# Add course list filter block to patterns

Implements https://github.com/Automattic/sensei/issues/5587

### Changes proposed in this Pull Request

* Added the Course List Filter block in the patterns

### Testing instructions

- Add a Course List block in any GB editor page
- Try choosing a pattern
- Make sure both patterns show up with 3 filters on top
- Make sure they stay after selecting
- Make sure they render as expected on the frontend
